Security (Is it safe?)/Data security/Vulnerability disclosure program.yaml

```diff
@@ -9,6 +9,13 @@ criterias:
       - indicator: >+
           The company has a mechanism (ex: a bug bounty program) through which security researchers can
           submit vulnerabilities they discover.
+          
+          A company with a bug bounty program has an alternative mechanism to report vulnerabilities that non-technical 
+          consumers can use to report vulnerabilities.
+          
+          The company provides a clear description with examples of what is defined as a security vulnerability.
```

Comment:
We probably don't want companies to define security vulnerability. So, security researchers can report any finding or concern they have to companies.  @billfitzg @mrerecich @stephtngu-CR